.github/workflows/stale.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/stale@v1
+    - uses: actions/stale@v3.0.19
```

Comment:
Use `v3` instead of the full version and it should use the latest v3 available.

Reply:
done